__tests__/app.test.js

```diff
@@ -263,5 +263,21 @@ describe("PATCH REQUEST/api/articles/:article_id", () => {
           "bad request: not a valid vote input, try again"
         );
       });
-  });
-});
+  })
+})
+describe("CORE: DELETE /api/comments/:comment_id", ()=>{
+  test("204: deleting a comment with a given id", ()=>{
+    return request(app)
+    .delete('/api/comments/1')
+    .expect(204)
+    })
+
+  test("404: if the id doesnt exist or in this case has already been deleted", ()=>{
+    return request(app)
+    .delete('/api/comments/1')
+    .expect(404)
+    .then((response)=>{
+      expect(response.body.msg).toBe("Sorry, but this Id doesn't exist.") 
+    })
+    })
```

Comment:
this shouldn't work - the seed should be running before every test - change your beforeall to a beforeEach. then change this test to be an article id that does not exist 

you can also add a test for an invalid article_id here!